spec/pb_kits/playbook/kits/title_spec.rb

```diff
@@ -14,7 +14,7 @@
     .with_default("h3")
      }
   it { is_expected.to define_enum_prop(:variant)
-     .with_values(nil, "primary")
+     .with_values(nil, "link")
```

Comment:
Why is this spec changing in this PR. This seems out of place.

Reply:
This was changed because of the changed to the Title kit made by Jason last Friday.

Reply:
Ok. I think you may need to rebase off of `6.1.0` then to make this change disappear from the PR.